app/controllers/missions_controller.rb

```diff
@@ -72,10 +72,13 @@ def generate(mission)
   end
 
   def update_transaction
-    Missions::UpdateTransaction.new.with_step_args(
-      transform_time_slots_in_time_params_for_enrollment: [regulated: @mission.regulated?],
-      update: [mission: @mission]
-    ).call(permitted_params)
+    @update_transaction ||=
+      begin
```

Comment:
same here, why `begin`?

Reply:
see https://www.justinweiss.com/articles/4-simple-memoization-patterns-in-ruby-and-one-gem/
in multi-line memoization section